### PR TITLE
[Content migration] - HRMP Channels

### PR DESCRIPTION
**Pull request by 0xLucca**
This PR incorporates content from the polkadot-ecosystem-docs-draft repository.

Migrated content:
- HRMP Channels

### CODE REVIEW

**Comment by 0xLucca** [comment]
@eshaben Please check if this section is located where expected. It is slightly different from what we have in the IA sheet. Develop > Parachain Developers > Interoperability > Open Message Passing Channels.

**Comment by eshaben** [comment]
@0xLucca all good from my end on the AI. We'll revisit later as needed, but I think it's good for now 👍

**Review by eshaben — APPROVED** [review]
LGTM! The JSON file needs formatting, but that'll get fixed automatically once the husky pre-commit hook is in and working 🙂

File: develop/parachain-devs/interoperability/hrmp-channels/para-to-para.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+An additional HRMP channel must be established in the opposite direction to enable bidirectional communication. This requires repeating the request and acceptance process but with the parachains reversing their roles.
+
+Once both unidirectional channels are established, the parachains can send messages back and forth freely through the bidirectional HRMP communication channel.
+
+## Prerequisites
+
+Before proceeding, ensure you meet the following requirements:
+
+- Blockchain network with a relay chain and at least two connected parachains
```
Comment (CrackTheCode016) [inline develop/parachain-devs/interoperability/hrmp-channels/para-to-para.md:25]: Maybe seems a little odd to say "Blockchain network", maybe just network? Or Polkadot SDK-based network (sorry for the very picky nit!)

**Pull request merged**